src/app/services/mail/mail.service.ts

```diff
@@ -779,6 +780,47 @@ export class MailService {
       })
   }
 
+  /**
+   * Sends a payment confirmation to a valid email
+   * @param recipient the recipient email address
+   * @param formTitle the form title of the payment form
+   * @param responseId the response ID
+   * @param formId the payment form ID
+   * @param paymentId the payment ID
+   * @throws error if mail fails, to be handled by the caller
+   */
+  sendPaymentConfirmationEmail = ({
+    recipient,
+    formTitle,
+    responseId,
+    formId,
+    paymentId,
+  }: {
+    recipient: string
+    formTitle: string
+    responseId: string
+    formId: string
+    paymentId: string
+  }): ResultAsync<true, MailSendError> => {
+    const mail: MailOptions = {
+      to: recipient,
+      from: this.#senderFromString,
+      subject: `Your payment on ${this.#appName} was successful`,
+      html: generatePaymentConfirmationHtml({
+        appName: this.#appName,
+        formTitle,
+        responseId,
+        invoiceUrl: `${
+          this.#appUrl
+        }/api/v3/payments/${formId}/${paymentId}/invoice/download`,
```

Comment:
Possible to re-use `getPaymentInvoiceDownloadUrl from "frontend/src/features/public-form/utils/urls.ts"`?

Reply:
oh yeah! I'll move that into `shared`

Reply:
done! [fe366e8](https://github.com/opengovsg/FormSG/pull/6145/commits/fe366e8f60a8b2a41b95fd91de0454b11c29ac0d), [3b0ecd4](https://github.com/opengovsg/FormSG/pull/6145/commits/3b0ecd45457b9f03818a0b59ad1b219a562aabf8) and [cc08535](https://github.com/opengovsg/FormSG/pull/6145/commits/cc0853520c3bc57a71113175ab50ec94f79f7526)